src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Colours;
+import java.text.DecimalFormat;
+
+public class Circle extends Figure {
+    private static final double numberPi = Math.PI;
+    private static final int CONST_RADIUS = 10;
+    private double radius = 0;
+
+    public Circle() {
+        super(Colours.WHITE.name());
+        this.radius = CONST_RADIUS;
```

Comment:
A constructor must not contain any logic. It shouldn't be responsible for settings default values